05-onchain.md

```diff
@@ -185,231 +161,193 @@ Similarly, when node A sees a *commitment transaction* from B:
 
 
 A node MUST handle the broadcast of any valid *commitment transaction*
-from B in this way; if it is unable to do so it MUST warn about lost funds.
-
+from node B in this way; if it is unable to do so it MUST warn about lost funds.
 
 ## Rationale
 
-
 Spending the `to_local` output avoids having to remember the complicated
 witness script associated with that particular channel for later
 spending.
 
-
 Note that there can be more than one valid, unrevoked *commitment
 transaction* after a signature has been received via `commitment_signed` and
-before the corresponding `revoke_and_ack`.  Either commitment can serve as
-B's *commitment transaction*, hence the requirement to handle both.
+before the corresponding `revoke_and_ack`. Either commitment can serve as
+a node's *commitment transaction*, hence the requirement to handle both.
 
-
-In the case of data loss, a node can reach a state where we don't
-recognize all of B's commitment transaction HTLC outputs.  It can tell
+In the case of data loss, a node can reach a state where it doesn't
+recognize all of the other node's commitment transaction HTLC outputs. It can tell
 this has happened because the commitment number will be greater than
-expected, and the fact that the transaction has been signed by this
-node.  If both nodes support `option-data-loss-protect` the node will
-know the B's `per_commitment_point` and thus be able to derive its own
-`remotekey` for the transaction and salvage its own funds (but not
+expected, and because it has signed the transaction.
+If both nodes support `option-data-loss-protect` the node will
+know B's `per_commitment_point` and thus be able to derive its own
+`remotekey` for the transaction and salvage its own funds (but not its
 HTLCs).
 
-
 # On-chain HTLC Output Handling: Our Offers
 
+Each HTLC output can only be spent by us, as the payer, after it's timed out,
+or by them, as the recipient, if they have the payment preimage.
```

Comment:
Offer/receive is the general usage elsewhere (eg title).  Payer is a bit confusing when we zoom out: if we're routing payments for others, though TBH same problem with recipient. 

It would be offer/accept, but the protocol doesn't actually allow you to *refuse*, you can just fail once the offer has been locked into both sides, so the word accept would be misleading.

As you can see from the text, I struggled with this at different times, so not sure I have anything concrete to add..

Reply:
I've added offerer/payer and recipient/? as something to consider when we're done, for overall consistent. And maybe something to add to the glossary.

Payer is used in some places, but maybe it's only used when it's the person offering the original contract? I dunno. All added to me overall considerations.

In this case, I've changed "payer" to "offerer", which was the other option I was considering. It may be moot, per a further discussion.